src/sdk/base/mediastream-factory.js

```diff
@@ -147,146 +147,86 @@ export class MediaStreamFactory {
    */
   static createMediaStream(constraints) {
     if (typeof constraints !== 'object' ||
-        (!constraints.audio && !constraints.video)) {
+      (!constraints.audio && !constraints.video)) {
       return Promise.reject(new TypeError('Invalid constrains'));
     }
     if (!isVideoConstrainsForScreenCast(constraints) &&
-        (typeof constraints.audio === 'object') &&
-        constraints.audio.source ===
-            MediaFormatModule.AudioSourceInfo.SCREENCAST) {
+      (typeof constraints.audio === 'object') &&
+      constraints.audio.source ===
+      MediaFormatModule.AudioSourceInfo.SCREENCAST) {
       return Promise.reject(
-          new TypeError('Cannot share screen without video.'));
+        new TypeError('Cannot share screen without video.'));
     }
     if (isVideoConstrainsForScreenCast(constraints) && !utils.isChrome() &&
-        !utils.isFirefox()) {
+      !utils.isFirefox()) {
       return Promise.reject(
-          new TypeError('Screen sharing only supports Chrome and Firefox.'));
+        new TypeError('Screen sharing only supports Chrome and Firefox.'));
     }
     if (isVideoConstrainsForScreenCast(constraints) &&
-        typeof constraints.audio === 'object' &&
-        constraints.audio.source !==
-            MediaFormatModule.AudioSourceInfo.SCREENCAST) {
+      typeof constraints.audio === 'object' &&
+      constraints.audio.source !==
+      MediaFormatModule.AudioSourceInfo.SCREENCAST) {
       return Promise.reject(new TypeError(
-          'Cannot capture video from screen cast while capture audio from'
-          + ' other source.'));
+        'Cannot capture video from screen cast while capture audio from'
+        + ' other source.'));
     }
-    // Screen sharing on Chrome does not work with the latest constraints
-    // format.
-    if (isVideoConstrainsForScreenCast(constraints) && utils.isChrome()) {
-      if (!constraints.extensionId) {
-        return Promise.reject(new TypeError(
-            'Extension ID must be specified for screen sharing on Chrome.'));
-      }
-      const desktopCaptureSources = ['screen', 'window', 'tab'];
-      if (constraints.audio) {
-        desktopCaptureSources.push('audio');
+
+    // check and convert constraints
```

Comment:
Start with capital letter and end with period.

Reply:
Don't got its meaning